scripts/archlinux/solidity-git/PKGBUILD

```diff
@@ -0,0 +1,65 @@
+# Maintainer: Afri 5chdn <aur@cach.co>
+# Available from AUR: https://aur.archlinux.org/packages/solidity-git/
+
+pkgname=solidity-git
+pkgver=0.4.3.git.20161020.9d30450
```

Comment:
This is the way the nightly versions are determined for Emscripten: https://github.com/ethereum/solidity/blob/develop/scripts/travis-emscripten/publish_binary.sh#L33-L40